README.md

```diff
@@ -1,2 +1,43 @@
 # RoboTalk
 An Android app to convert text to speech and vice versa.
+
+## Features
```

Comment:
Can you add a section of "Getting started" that tells how to set the development environment for the contributors to work on the project.

Reply:
@ShivangiSingh17 added the "Getting Started" Section.
please review and suggest any changes applicable.